src/variants/lambdatreasury/storage.mligo

```diff
@@ -0,0 +1,185 @@
+// SPDX-FileCopyrightText: 2022 Tezos Commons
+// SPDX-License-Identifier: LicenseRef-MIT-TC
+
+#if !VARIANT_STORAGE
+#define VARIANT_STORAGE
+
+#include "variants/lambda/types.mligo"
+#include "error_codes.mligo"
+
+// Xtz transfer amount cannot be 0
+let zero_mutez_err_msg = "ZERO_MUTEZ"
+
+// Xtz transfer amount cannot be smaller than `min_xtz_amount`
+let too_small_xtz_err_msg = "LOW_XTZ"
+
+// Xtz transfer amount cannot be bigger than `max_xtz_amount`
+let too_large_xtz_err_msg = "HIGH_XTZ"
```

Comment:
Why not import these from `src/common/errors.mligo`?

Likewise in `lambdaregistry`

Reply:
This appear to cause circular imports because `common/errors.mligo` import `types.mligo` and `types.mligo` has to import `storage.mligo` for the variant being built.